webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart

```diff
@@ -200,24 +201,25 @@ class StandardMessagesManager {
 
       if (localMessage == null || storageMessage == null) {
         // todo: the message has been deleted from the UI or storage
-      } else if (localMessage.categoryName != storageMessage.categoryName) {
+      } else if (localMessage.categoryName != storageMessage.categoryName && localMessage.categoryName != standardMessage.categoryName) {
         // the category has been updated
         _view.categoriesById[standardMessage.categoryId].showAlternative(standardMessage.categoryName);
-      } else if (localMessage.groupName != storageMessage.groupName) {
+      } else if (localMessage.groupName != storageMessage.groupName && localMessage.groupName != standardMessage.groupName) {
         // the group has been updated
          _view.categoriesById[standardMessage.categoryId].groupsById[standardMessage.groupId].showAlternative(standardMessage.groupName);
-      } else if (localMessage.text != storageMessage.text || localMessage.translation != storageMessage.translation) {
+      } else if ((localMessage.text != storageMessage.text && localMessage.text != standardMessage.text) || (localMessage.translation != storageMessage.translation && localMessage.translation != standardMessage.translation)) {
```

Comment:
let's split this on two lines to it reads a bit easier
```suggestion
      } else if ((localMessage.text != storageMessage.text && localMessage.text != standardMessage.text)
             || (localMessage.translation != storageMessage.translation && localMessage.translation != standardMessage.translation)) {
```

Reply:
Fixed!